templates/listing_table.tpl

```diff
@@ -33,6 +34,7 @@
         {/if}
         </td>
     {/foreach}
+        <td><a href="index.php?page=gallery&dn={$entry.dn|escape:'url'}">{$msg_gallery}</a></td>
```

Comment:
Maybe we should use a GET parameter named "groupdn" to clearly says that we are displaying a gallery for a group.